diff --git a/node_modules/material-table/dist/material-table.js b/node_modules/material-table/dist/material-table.js
index 03bfc7e..90d2fdc 100644
--- a/node_modules/material-table/dist/material-table.js
+++ b/node_modules/material-table/dist/material-table.js
@@ -800,7 +800,9 @@ var ScrollBar = function ScrollBar(_ref) {
   } else {
     return React.createElement("div", {
       style: {
-        overflowX: 'auto'
+        overflowX: 'auto',
+        marginLeft: '3em',
+        marginRight: '3em'
       }
     }, children);
   }
